pipelined/bevy_render2/src/mesh/mesh/mod.rs

```diff
@@ -622,15 +622,15 @@ impl RenderAsset for Mesh {
         let vertex_buffer_data = mesh.get_vertex_buffer_data();
         let vertex_buffer = render_device.create_buffer_with_data(&BufferInitDescriptor {
             usage: BufferUsages::VERTEX,
-            label: None,
+            label: Some("Mesh Vertex Buffer"),
             contents: &vertex_buffer_data,
         });
 
         let index_info = mesh.get_index_buffer_bytes().map(|data| GpuIndexInfo {
             buffer: render_device.create_buffer_with_data(&BufferInitDescriptor {
-                usage: BufferUsages::INDEX,
+                usage: BufferUsages::INDEX | BufferUsages::MAP_WRITE,
```

Comment:
This usage removes the warning in Firefox, and makes it crash a little further away on Safari.

I don't really know the impact of adding it though, is it an issue to always add it? Should it be under the `webgl` feature?

Reply:
removed with wgpu update